tests/libnet/hotplug.go

```diff
@@ -152,21 +130,7 @@ func interfaceStatusFromInterfaceNames(queueCount int32, ifaceNames ...string) [
 	return ifaceStatus
 }
 
-func newVMWithOneInterface() *v1.VirtualMachine {
-	vm := libvmi.NewVirtualMachine(libvmifact.NewAlpineWithTestTooling(), libvmi.WithRunning())
-	vm.Spec.Template.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
-	vm.Spec.Template.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultMasqueradeNetworkInterface()}
-	return vm
-}
-
-func migrate(vmi *v1.VirtualMachineInstance) {
-	By("migrating the VMI")
-	migration := libmigration.New(vmi.Name, vmi.Namespace)
-	migrationUID := libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
-	libmigration.ConfirmVMIPostMigration(kubevirt.Client(), vmi, migrationUID)
-}
-
-func patchVMWithNewInterface(vm *v1.VirtualMachine, newNetwork v1.Network, newIface v1.Interface) error {
+func PatchVMWithNewInterface(vm *v1.VirtualMachine, newNetwork v1.Network, newIface v1.Interface) error {
 	patchData, err := patch.GeneratePatchPayload(
```

Comment:
Possibly migrate this to `patch.PatchSet` in a follow up?

Reply:
Right, I will send a new PR for this one.

Reply:
ACK